### PR TITLE
Feature/ui packaging

### PR DESCRIPTION
**Pull request by sreevatsanraman**
- Packaging RPM and debian
  - Enable new UI by setting the env variable ENABLE_BETA_UI=true

### CODE REVIEW

**Comment by sreevatsanraman** [comment]
Verification:
a) If env variable is not set or if it is set to false. Then start old UI
> env  | grep ENABLE_BETA_UI
>  /etc/init.d/cdap-web-app start
Thu Mar 19 23:14:55 UTC 2015 Starting web-app service on sree-test1107-1000.dev.continuuity.net
> ps -aef | grep node | grep web-app
cdap     21812     1  5 23:14 ?        00:00:00 node /opt/cdap/web-app/server/main.js
cdap     21840 21812  8 23:14 ?        00:00:00 /usr/bin/node /opt/cdap/web-app/server/main.js
cdap     21841 21812  9 23:14 ?        00:00:00 /usr/bin/node /opt/cdap/web-app/server/main.js

> env  | grep ENABLE_BETA_UI
ENABLE_BETA_UI=false
> /etc/init.d/cdap-web-app start
Thu Mar 19 23:15:40 UTC 2015 Starting web-app service on sree-test1107-1000.dev.continuuity.net
>  ps -aef | grep node | grep web-app
cdap     22068     1  9 23:15 ?        00:00:00 node /opt/cdap/web-app/server/main.js
cdap     22096 22068 17 23:15 ?        00:00:00 /usr/bin/node /opt/cdap/web-app/server/main.js
cdap     22097 22068 16 23:15 ?        00:00:00 /usr/bin/node /opt/cdap/web-app/server/main.js

b) if env variable set to true. start new UI
> env  | grep ENABLE_BETA_UI
ENABLE_BETA_UI=true
> /etc/init.d/cdap-web-app start
Thu Mar 19 23:16:57 UTC 2015 Starting web-app service on sree-test1107-1000.dev.continuuity.net
> ps -aef | grep node | grep web-app
cdap     22322     1 12 23:16 ?        00:00:00 node /opt/cdap/web-app/beta/server.js

File: cdap-ui/pom.xml [hunk 1]
```diff
@@ -118,6 +118,16 @@
             <artifactId>exec-maven-plugin</artifactId>
             <executions>
               <execution>
+              <!--Skipping the rpm package phase for cdap-ui. It will be run from cdap-web-app-->
+              <id>rpm-package</id>
+                <phase/>
```
Comment (albertshau) [inline cdap-ui/pom.xml]: nit: indentation

File: cdap-ui/pom.xml [hunk 2]
```diff
@@ -118,6 +118,16 @@
             <artifactId>exec-maven-plugin</artifactId>
             <executions>
               <execution>
+              <!--Skipping the rpm package phase for cdap-ui. It will be run from cdap-web-app-->
+              <id>rpm-package</id>
```
Comment (albertshau) [inline cdap-ui/pom.xml]: nit: indentation

**Comment by albertshau** [comment]
:+1:

File: cdap-ui/pom.xml (hunk 2 above)
Comment (sreevatsanraman) [inline cdap-ui/pom.xml]: Fixed.

**Pull request merged**